run_nextflow_mip.sh

```diff
@@ -38,11 +38,11 @@ module load Java/1.8.0_60
 if [ \$? -eq 0 ]; then
     echo "Nextflow done."
 
-    echo "Running Nextflow clean"
-    /hpc/diaggen/software/tools/nextflow clean -f -k -q
-
     echo "Zip work directory"
-    zip -r -m -q work.zip work
+    find work -type f | egrep "\.(command|exitcode)" | zip -@ -q work.zip
+
+    echo "Remove work directory"
+    rm -r work
```

Comment:
first the work directory was zipped, now removed. is there a reason for this?

Reply:
Yes. The work directory contains all input, outputs and command files of tasks. This is quite a large directory.
The find command on line 42 searches for relevant files and creates a zip with the things that need to be retained. 
Afterwards, the directory is removed.

Reply:
:+1: